.github/workflows/mirror-to-gh-pages.yml

```diff
@@ -15,11 +17,18 @@ permissions:
 jobs:
   mirror:
     runs-on: ubuntu-latest
+    # Only run if the triggering workflow succeeded (when triggered by workflow_run)
+    if: ${{ github.event_name == 'workflow_dispatch' || github.event.workflow_run.conclusion == 'success' }}
     steps:
       - name: Get release info
         id: release
         run: |
-          TAG="${{ github.event.inputs.tag || github.event.release.tag_name }}"
+          if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
+            TAG="${{ github.event.inputs.tag }}"
+          else
+            # Get the latest release tag when triggered by workflow_run
+            TAG=$(gh api "repos/${{ github.repository }}/releases/latest" | jq -r '.tag_name')
```

Comment:
or we could just make the tag mandatory in the workflow_dispatch 🤔

Reply:
yep, the cons that I see is that we manually need to set the tag but we already know that we want to do it for the latest one